src/tools/cephfs/shell/cephfs-shell

```diff
@@ -15,13 +15,16 @@ import re
 import shlex
 import stat
 import errno
+import distro
 
 from cmd2 import Cmd
 from cmd2 import __version__ as cmd2_version
 from packaging.version import Version
 
 # XXX: In cmd2 versions < 1.0.1, we'll get SystemExit(2) instead of
 # Cmd2ArgparseError
+if distro.name() == "Ubuntu" and distro.version() == "22.04":
+    cmd2_version = "2.3.3"
```

Comment:
This is a bit bothersome -- hardcoding a version. Any possibility to use a flag to force a certain call to be used?

Reply:
Yeah, agree!! I have pushed a new patch, how about this?